polaris/tasks/ocean/seamount/seamount.cfg

```diff
@@ -0,0 +1,89 @@
+# Options related to the vertical grid
+[vertical_grid]
+
+# Depth of the bottom of the ocean (m)
+bottom_depth = 5000.0
+
+# Number of vertical levels
+vert_levels = 10
+
+# The type of vertical grid
+grid_type = uniform
+
+# The type of vertical coordinate (e.g. z-level, z-star)
+coord_type = sigma
+
+# Whether to use "partial" or "full", or "None" to not alter the topography
+partial_cell_type = None
+
+# Options related to the seamount case
+[seamount]
+
+# Timestep per km horizontal resolution (s)
+dt_per_km = 10.
+
+# Barotropic timestep per km horizontal resolution (s)
+btr_dt_per_km = 2.5
+
+# The width of the domain in the across-slope dimension (km)
+ly = 320
+
+# The length of the domain in the along-slope dimension (km)
+lx = 320
+
+# Distance from two cell centers (km)
+resolution = 6.7
+
+# Bottom depth at bottom of seamount
+max_bottom_depth = ${vertical_grid:bottom_depth}
+
+# Logical flag that controls how the vertical profile of tracers.  See Beckmann and Haidvogel 1993 eqn 15-16 (unitless)
+# possible_values="linear, exponential"
+seamount_stratification_type = exponential
+
+# Density coefficient for linear vertical stratification (kg m^{-3})
+seamount_density_coef_linear = 1024.0
+
+# Density coefficient for exponential vertical stratification (kg m^{-3})
+seamount_density_coef_exp = 1028.0
+
+# Density gradient for linear vertical stratification, $\Delta_z \rho$ in Beckmann and Haidvogel eqn 15 (kg m^{-3})
+seamount_density_gradient_linear = 0.1
+
+# Density gradient for exponential vertical stratification, $\Delta_z \rho$ in Beckmann and Haidvogel eqn 16 (kg m^{-3})
+seamount_density_gradient_exp = 3.0
+
+# Density reference depth for linear vertical stratification (m)
+seamount_density_depth_linear = 4500.0
+
+# Density reference depth for exponential vertical stratification (m)
+seamount_density_depth_exp = 500.0
+
+# Density reference for eos to initialize temperature (kg m^{-3})
+seamount_density_ref = 1028.0
+
+# Reference temperature for eos to initialize temperature (C)
+seamount_density_Tref = 5.0
+
+# Linear thermal expansion coefficient to initialize temperature (kg m^{-3} C^{-1})
+seamount_density_alpha = 0.2
```

Comment:
These settings should either be removed and applied directly in a new [ocean] section https://github.com/E3SM-Project/polaris/blob/705945229c4fdb933352b5449b35a101754a4bce/polaris/ocean/ocean.cfg#L19-L32

or you can add the [ocean] section, keep these options here, and use references, e.g.,
`eos_linear_alpha = ${seamount:seamount_density_alpha}`

Reply:
Done. It's slick how you can specify [ocean] parameters from the defaults in `ocean.cfg`, or specify them for each test case in the individual `*.cfg`!